jobs/delorean/2.0/ews/3scale.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- job:
+    name: 3scale_brew_build_created
+    display-name: 3Scale Brew Build Created
+    description: |
+      <h2><a href="https://github.com/integr8ly/ci-cd/tree/master/jobs/delorean/2.0/ews">Click here to see the job definition on GitHub!</a></h2>
+
+      <h3>This job gets triggered by Brew Builds on the UMB</h3>
+    disabled: false
+    project-type: pipeline
+
+    # Trigger responds to new builds of 3scale-operator-metadata-container
+    # e.g. https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=74069
+    triggers:
+      - ci-trigger:
+          override-topic: Consumer.rh-jenkins-ci-plugin.87a18ae3-51c3-495f-834a-a6edd86e1435.VirtualTopic.eng.brew.build.complete 
+          message-checks:
+            - field: '$.info.package_id'
+              value: 74069
+
+    parameters:
+    - string:
+          name: CI_MESSAGE
+          description: This will be populated by the CI trigger
+
+    pipeline-scm:
+      script-path: jobs/delorean/2.0/ews/Jenkinsfile
+      scm:
+      - git:
+          branches:
+          - master
+          url: git@github.com:integr8ly/ci-cd.git
+          credentials-id: jenkins-github-api-token
+          skip-tag: true
+          wipe-workspace: false
```

Comment:
![image](https://user-images.githubusercontent.com/442386/74243859-633fe480-4cd8-11ea-8766-fa907a436b28.png)

When you see this at the end of a line, it's GitHub telling you that there's no newline set on the last line. This is not often a problem, but sometimes is -- yesterday someone incorrectly `git blame`d a line change to me and was asking me for context, even though all I'd done was added another line after it, causing a `\n` to be added to the line in question.

If you want, you can probably get your editor to automatically make sure there's a newline at the end.